app/controllers/schools/dashboard_controller.rb

```diff
@@ -4,5 +4,24 @@ class Schools::DashboardController < Schools::BaseController
   skip_after_action :verify_authorized
   skip_after_action :verify_policy_scoped
 
-  def show; end
+  def show
+    @school = current_user.induction_coordinator_profile.schools.first
+
+    @cohorts = [Cohort.current].map do |cohort|
```

Comment:
Why are we looping when there's only 1 current cohort?

Reply:
Futureproofing! So that the view won't need updating next year.

Reply:
The way I understand it. There will always be one current cohort. So when 2022 becomes the new cohort 2021 will cease to be current.